crates/bdk/src/wallet/coin_selection.rs

```diff
@@ -1411,34 +1421,6 @@ mod test {
         }
     }
 
-    #[test]
-    fn test_single_random_draw_function_success() {
```

Comment:
Maybe worth keeping a version of this test to be sure `single_random_draw` still works as intended.